Scripts/script-Set.yml

```diff
@@ -5,7 +5,20 @@ name: Set
 script: |-
   var ec = {};
   ec[args.key] = args.value;
-  return {Type: entryTypes.note, Contents: ec, ContentsFormat: formats.json, HumanReadable: 'Key ' + args.key + ' set', EntryContext: ec};
+  var result = {
+      Type: entryTypes.note,
+      Contents: ec,
+      ContentsFormat: formats.json,
+      HumanReadable: 'Key ' + args.key + ' set'
+  };
+
+  if (args.append) {
+      result.EntryContext = ec;
+  } else if (!args.append || args.append === 'false') {
```

Comment:
In case append was not passed or false string was passed. 
But I need to switch the order between them, in case someone pass `append=false`